beta-src/src/components/ui/WDMain.tsx

```diff
@@ -36,6 +39,8 @@ const WDMain: React.FC = function (): React.ReactElement {
   const status = useAppSelector(gameStatus);
   const data = useAppSelector(gameData);
   const maps = useAppSelector(gameMaps);
+  const legalOrders = useAppSelector(gameLegalOrders);
+  const gameDataResponse = useAppSelector(gameData);
```

Comment:
This is redundant with  `const data = useAppSelector(gameData);` a few lines above?

Reply:
Good catch, I didn't notice